### PR TITLE
9249 Registrations Schema

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#9249

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/schemas/party.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
                     "title": "The type of party",
                     "enum": [
                         "organization",
-                        "person"
+                        "person",
+                        "businessOrCorporation"
```
Comment (thorwolpert) [inline src/registry_schemas/schemas/party.json:16]: organization covers this, or at least that is the intent.
people or org... or you could call is a business if you wanted to change it everywhere.

Comment (lekshmimallika-aot) [inline src/registry_schemas/schemas/party.json:16]: Is this new type needed? Can't organization be used?

File: src/registry_schemas/schemas/party.json [hunk 2]
```diff
@@ -34,6 +35,15 @@
                 "email": {
                     "type": "string",
                     "format": "email"
+                },
+                "businessNumber": {
```
Comment (thorwolpert) [inline src/registry_schemas/schemas/party.json:39]: this is the taxId, why wouldn't it be held at the business level?

**Review by thorwolpert — COMMENTED** [review]
need to chat

File: src/registry_schemas/schemas/party.json (hunk 1 above)
Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/party.json]: removed businessOrCorporation

Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/party.json]: removed businessOrCorporation

File: src/registry_schemas/schemas/registration.json [hunk 3]
```diff
@@ -32,6 +32,14 @@
                 "contactPoint": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
                 },
+                "naicsCode": {
```
Comment (thorwolpert) [inline src/registry_schemas/schemas/registration.json:35]: This doesn't follow the pattern of a block with all the info in it.
NAICS will also get used across all business types, so should be an easier block to add and maybe in the business block?

**Review by thorwolpert — COMMENTED** [review]
should naics follow the common pattern of being a block of info?

File: src/registry_schemas/schemas/registration.json (hunk 3 above)
Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/registration.json]: moved it

**Review by lekshmimallika-aot — APPROVED** [review]

**Pull request merged**